BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/IndoorPositioning.java

```diff
@@ -151,6 +151,9 @@ public static boolean unregisterLocationListener(LocationListener locationListen
 
             @Override
             public boolean matches(Beacon beacon) {
+                if (!(beacon instanceof IBeacon)) {
```

Comment:
As discussed, we should refactor this later on, because currently it is not restricted to IBeacons.

Reply:
Refactored